ui/apps/platform/cypress/integration/audit/listeningEndpoints/ListeningEndpoints.helpers.js

```diff
@@ -0,0 +1,10 @@
+import { visit } from '../../../helpers/visit';
+
+const basePath = '/main/audit/listening-endpoints/';
+
+export function visitListeningEndpoints() {
+    visit(basePath);
+
+    cy.get('h1:contains("Listening endpoints")');
+    cy.location('pathname').should('eq', basePath);
```

Comment:
I think asserting the path is useful when testing whether a nav item (like in the left menu) goes to to the proper place, but here it's just asserting that both Cypress (with `cy.visit`) and the browser are functioning.